app/controllers/provider_interface/courses/providers_controller.rb

```diff
@@ -0,0 +1,37 @@
+module ProviderInterface
+  module Courses
```

Comment:
could we rename this module grouping/wizard as well, we need something specific to indicate this is changing a course before an offer as course is a bit vague, what do you think?